internal/experiment/webconnectivityqa/httpdiff.go

```diff
@@ -92,7 +92,7 @@ func httpDiffWithInconsistentDNS() *TestCase {
 			TitleMatch:            false,
 			XStatus:               96, // StatusAnomalyHTTPDiff | StatusAnomalyDNS
 			XDNSFlags:             4,  // AnalysisDNSUnexpectedAddrs
-			XBlockingFlags:        35, // analysisFlagSuccess | analysisFlagDNSBlocking | analysisFlagTCPIPBlocking
+			XBlockingFlags:        17, // analysisFlagDNSBlocking | analysisFlagHTTPDiff
```

Comment:
This seems to be a regression that we have fixed. The correctly expected result is to see that there is DNS blocking (because we're spoofing the IP address) and HTTP diff (because we're injecting a response).

It should also be noted that this test case seems flaky when running it locally. Because there is a race condition involved in spoofing the webpage, we would probably need to increase the RTTs to make sure that the censor always wins the race with ordinary packets in flight.